### ISSUE TITLE
update table view

### ISSUE THREAD

**Issue by heliosdrm** [julia-vscode/julia-vscode#1525]
Either automatically, or clicking on a button in the table viewer, refresh the view to see changes since it was opened.

**Comment by davidanthoff** [comment]
I don't think this will be feasible, TBH. My current idea is that the table view shows a snapshot of the data as it existed when the `display` method was called. I think anything that resembles more of a live view would be way, way too brittle.

**Comment by heliosdrm** [comment]
That is very understandable. But do you think that there could be a feasible, intermediate option between a "live view" and a static snapshot, e.g. updating the snapshot on demand? My motivation for this FR is that if I open a view of a table and later I want to see a modified version, I have to close the first tab to avoid the confusing situation of having two tabs with different versions of the same table. That quickly becomes annoying if I want to do it repeatedly.

Would it be possible to detect that a table with the same variable name is already being viewed, and in that case overwrite the view on the same tab, instead of opening a new one? That would improve a lot the experience in that use case.

**Comment by davidanthoff** [comment]
I think even that is not trivial: we currently have no notion of which variable a table came from. It could have been:
```
julia> data |> @filter(_.foo="bar") |> vscodedisplay
```

**Comment by heliosdrm** [comment]
I understand. And what about recording that information in limited contexts, like when the "Open in VS Code" button in the workspace explorer is used? In such cases the variable name should always be available.

![image](https://user-images.githubusercontent.com/10651028/88769089-44e17e80-d17c-11ea-805c-2a76c052b128.png)
